KS.Fiks.Maskinporten.Client/MaskinportenClientConfiguration.cs

```diff
@@ -9,14 +11,25 @@ public class MaskinportenClientConfiguration
             string tokenEndpoint,
             string issuer,
             int numberOfSecondsLeftBeforeExpire,
-            X509Certificate2 certificate,
+            X509Certificate2 certificate = null,
+            RSA privateKey = null,
+            RSA publicKey = null,
+            string keyIdentifier = null,
             string consumerOrg = null)
         {
+            if (certificate == null && (privateKey == null || publicKey == null))
```

Comment:
Maybe add a check if both certificate and keys are provided and throw exception in that case also as one is supposed to provider just one of them